pages/common/jp2a.md

```diff
@@ -0,0 +1,28 @@
+# jp2a
+
+> Convert JPEG images to ASCII.
+> More information: <https://csl.name/jp2a/>.
+
+- Read JPEG image from a file and print in ASCII:
+
+`jp2a {{path/to/image.jpeg}}`
+
+- Read JPEG image from a URL and print in ASCII:
+
+`jp2a {{www.example.com/image.jpeg}}`
+
+- Colorize the ASCII output:
+
+`jp2a --colors {{path/to/image.jpeg}}`
+
+- Specify characters to be used for the ASCII output:
+
+`jp2a --chars={{"..-ooxx@@"}} {{path/to/image.jpeg}}`
```

Comment:
Oops! Sorry, I missed this one. I would suggest singlge quotes outside would aid clarity here. With double quotes if one were to use a dollar sign, it would be interpreted as a variable name by the shell.
```suggestion
`jp2a --chars='{{..-ooxx@@}}' {{path/to/image.jpeg}}`
```